src/SLCore.IntegrationTests/SLCore.IntegrationTests.csproj

```diff
@@ -13,4 +13,11 @@
     <ProjectReference Include="..\TestInfrastructure\TestInfrastructure.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Content Include="..\EmbeddedSonarAnalyzer.props">
+      <Link>EmbeddedSonarAnalyzer.props</Link>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
```

Comment:
Not sure this is the correct option. WDYT?

Reply:
I prefer to have it always. It's a tiny file, anyway. I don't think copying all the time is much more expensive than checking if the file is newer.